src/collection.js

```diff
@@ -502,9 +523,33 @@ export default class Collection {
           syncResultObject.add("published", published.map(res => res.data));
           return syncResultObject;
         });
-      });
+      })
+      // Handle conflicts, if any
+      .then(result => this._handleConflicts(result, options.strategy));
   }
 
+  /**
+   * Handles synchronization conflicts according to specified strategy.
+   *
+   * @param  {SyncResultObject} result    The sync result object.
+   * @param  {String}           strategy  The sync strategy.
+   * @return {Promise}
+   */
+  _handleConflicts(result, strategy=Collection.strategy.MANUAL) {
+    if (strategy === Collection.strategy.MANUAL) {
+      return Promise.resolve(result);
+    }
+    if (!result.conflicts.length) {
+      return Promise.resolve(result);
+    }
+    return Promise.all(result.conflicts.map(conflict => {
+      const resolution = strategy === Collection.strategy.CLIENT_WINS ?
+                         conflict.local : conflict.remote;
+      return this.resolve(conflict, resolution);
+    })).then(imports => {
+      return result.reset("conflicts");
```

Comment:
Added a `merged` entry to the result object in ff710bf82d0755f657c079118a1b88573b8ddc41.